src/firefly/python/display/firefly_client.py

```diff
@@ -157,7 +161,7 @@ def __init__(self, host=_my_localhost, channel=None):
         self.connect()
 
     def _handle_event(self, ev):
-        for callback, eventIDList in self.listeners.items():
+        for callback, eventIDList in list(self.listeners.items()):
```

Comment:
Here is a case where `futurize -2` is too conservative by wrapping  `self.listeners.items()` in a list. The `list()` can be removed.